adobe_vipm/flows/validation/transfer.py

```diff
@@ -69,37 +73,89 @@ def get_prices(order, commitment, adobe_skus):
         return get_prices_for_skus(product_id, currency, adobe_skus)
 
 
+def has_order_line_updated(order_lines, adobe_items, quantity_field):
+    """
+    Compare order lines and Adobe items to be transferred
+    Args:
+        order_lines (list): List of order lines
+        adobe_items (list): List of adobe items to be transferred.
+        quantity_field (str): The name of the field that contains the quantity depending on the
+        provided `adobe_object` argument.
+
+    Returns:
+        bool: True if order line is not equal to adobe items, False otherwise.
+
+    """
+    order_line_map = {
+        order_line["item"]["externalIds"]["vendor"]: order_line["quantity"]
+        for order_line in order_lines
+    }
+
+    adobe_items_map = {
+        get_partial_sku(adobe_item["offerId"]): adobe_item[quantity_field]
+        for adobe_item in adobe_items
+    }
+    return order_line_map != adobe_items_map
 
-def add_lines_to_order(mpt_client, order, adobe_object, commitment, quantity_field):
+
+def add_lines_to_order(mpt_client,
```

Comment:
Please, run black formatter, as I remember we have another rule to format function definition here